src/database/accounts.js

```diff
@@ -51,29 +51,16 @@ module.exports = {
     },
 
     /**
-     * Search for a user by name
+     * Search for a user by any field
      */
-    search: function (name, fields, callback) {
-        /* This bit allows it to accept varargs
```

Comment:
The last time `acp.js` was touched meaningfully (i.e. not just stripping comments or moving the file path) was 2014, so yeah that predates ES2015.

But if nothing is actually using the optionality of the fields, I think it's fine to not preserve that behavior and just make it a required field.  A quick grep shows nothing is using it except the ACP.